sw/simulator/nps/nps_fdm_jsbsim.cpp

```diff
@@ -510,14 +510,17 @@ static void init_jsbsim(double dt)
 
   char* pprz_home = getenv("PAPARAZZI_HOME");
 
+  int cnt = -1;
   if (strlen(pprz_home) < sizeof(buf)) {
-    sprintf(buf, "%s/conf/simulator/jsbsim/", pprz_home);
+    cnt = sprintf(buf,"%s/conf/simulator/jsbsim/", pprz_home);
```

Comment:
@flixr Try the snprintf - It was crashing the NPS for me, so maybe you'll be luckier.